doc/private_net_sync.rst

```diff
@@ -0,0 +1,306 @@
+Creating A Private Network With Two Syncing Nodes
+==================================================
+These instructions cover creating an Ethereum private network consisting of two nodes - one node will mine blocks and the other node will connect to the first node and sync the mined blocks to its own block database.
+
+
+What is a Private Network?
+--------------------------
+For the purposes of this documentation, a **private network** can be thought of as a network of Ethereum nodes only accessible on your physical machine.
+
+What is a Private Chain?
```

Comment:
Yes, I meant to have both sections since I think a private network and a private chain are 2 distinct concepts (though they are frequently used together) - for example, you can run a private network with a mainnet chain config.

I've tweaked the verbiage in the "private chain" section so that it doesn't sound so similar to the previous section.